### PR TITLE
Add unwrapElement method to UpcastWriter

### PR DESCRIPTION
**Pull request by msamsel**
### Suggested merge commit message ([convention](https://github.com/ckeditor/ckeditor5-design/wiki/Git-commit-message-convention))

Other: Add unwrapElement method to UpcastWriter.

---

### Additional information

PR extracted from ckeditor/ckeditor5-paste-from-office#69`

### CODE REVIEW

File: src/view/upcastwriter.js [hunk 1]
```diff
@@ -172,6 +172,27 @@ export default class UpcastWriter {
...
+	unwrapElement( element ) {
+		const parent = element.parent;
+
+		if ( parent ) {
+			const index = parent.getChildIndex( element );
+
+			this.remove( element );
+			this.insertChild( index, element.getChildren(), parent );
+
+			return true;
```
Comment (jodator) [inline src/view/upcastwriter.js]: The method should not return true/false - we don't need that.

Check the model's `writer.unwrap()` how we handle this: https://github.com/ckeditor/ckeditor5-engine/blob/ad9159ca794096cbed1ab44180df0c74181d2f31/src/model/writer.js#L851-L861

**Comment by jodator** [comment]
After some clarification from @msamsel I see that some methods of upcast writer return true/false in similar scenarios but TBH I don't see why. So I'm for not returning anything but adding a `if( element.parent )` check to not perform unwrap on detached elements.

File: src/view/upcastwriter.js [hunk 2]
```diff
@@ -172,6 +172,24 @@ export default class UpcastWriter {
...
 	}
 
+	/**
+	 * Removes given element from view structure and places its children in its position.
+	 * It does nothing if element has no parent.
+	 *
+	 * @param {module:engine/view/element~Element} element Element which will be unwrapped
+	 * @returns {Booolean} Whether element was successfully unwrapped.
+	 */
+	unwrapElement( element ) {
```
Comment (oskarwrobel) [inline src/view/upcastwriter.js]: Just for my curiosity. Why not just `unwrap`? Other methods from this class (like `remove`, `replace`, `rename`) don't have `Element` suffix.

Comment (msamsel) [reply src/view/upcastwriter.js]: I've followed byt this explanaition:
https://github.com/ckeditor/ckeditor5-paste-from-office/pull/72#discussion_r310022092

**Pull request merged**